src/WebJobs.Script/Content/Script/res.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+module.exports = () => {
+    var res = {};
+
+    res.status = res.sendStatus = (statusCode) => {
+        res.statusCode = statusCode;
+        return res;
+    };
+
+    res.send = res.json = (body) => {
```

Comment:
@mamaso , I might be missing your point, but I'm confused about this statement:

>... I should set content-type to `application/json` anyway and then the c# webapi stuff can do its content negotiation.

We need to set the content type to `application/json` if that content type is what is expected when using express and `res.json`, and in that case, there would be **no conneg**. The runtime would honor the `content-type` set in the function and that would be used no matter what the `Accept` header stated.

If, on the other hand, we do not set the `content-type` header when using `res.json` (and the others mentioned by Mathew above), then we would end up relying on conneg and the result *could* be unexpected if the intention is to match the express behavior (as what the client specifies in the `accept` header would be evaluated and you could end up with a different `content-type`)

This is unrelated to your changes, but before commenting I ran a few tests with our current logic that would trigger content negotiation in Node and we have some issues we need to address. We can chat with the larger group tomorrow and open issues to document them.